### PR TITLE
Document num_cached_file_handles configuration property

### PR DESCRIPTION
**Pull request by zhli1142015**

### CODE REVIEW

File: velox/docs/configs.rst [hunk 1]
```diff
@@ -389,6 +389,10 @@ Hive Connector
      - integer
      - 128MB
      - Maximum distance in bytes between chunks to be fetched that may be coalesced into a single request.
+   * - num_cached_file_handles
+     - integer
+     - 20000
+     - Maximum number of entries in the file handle cache.
```
Comment (mbasmanova) [inline velox/docs/configs.rst:395]: Let's document the meaning of zero and negative values.

Comment (zhli1142015) [reply velox/docs/configs.rst]: Added, thanks.

File: velox/docs/configs.rst [hunk 2]
```diff
@@ -389,6 +389,11 @@ Hive Connector
      - integer
      - 128MB
      - Maximum distance in bytes between chunks to be fetched that may be coalesced into a single request.
+   * - num_cached_file_handles
+     - integer
+     - 20000
+     - Maximum number of entries in the file handle cache. The value must be non-negative. Value zero
```
Comment (mbasmanova) [inline velox/docs/configs.rst:395]: > means no cache eviction

Not sure if readers will understand this. Perhaps, "Zero value indicates infinite cache capacity."

Comment (zhli1142015) [reply velox/docs/configs.rst]: Make sense, updated.

**Review by mbasmanova — APPROVED** [review]
Thanks.

**Pull request closed without merging**